.github/workflows/fqdn-perf.yaml

```
@@ -108,6 +106,7 @@ jobs:
             --set pprof.enabled=true \
             --helm-set=prometheus.enabled=true \
             --helm-set=dnsProxy.proxyResponseMaxDelay=100s \
+            --helm-set=cluster.name=${{ env.cluster_name }} \
```

```suggestion
            --helm-set=cluster.name=${{ env.test_name }} \
```

Nit. The Cilium cluster name does not need to be unique, and the test name looks a bit more user-friendly. Certainly not blocking (in case you decide to change it, the same would apply for the other occurrences).

---

I would leave `run_id`. If you ever download sysdump only, it's easier to pin point to specific action run. But I don't have a strong opinion, we can switch it later if we think it makes more sense.